Update sql-traits.md Add Troubleshooting step (#5453)
segmentio/segment-docs: src/unify/Traits/sql-traits.md

### Why doesn't the identifier updated by a SQL trait show the correct value found in the column?

Ensure that the name given to the SQL trait is not the same name as the identifier or column name from the query. To use SQL traits to update an identifier, the identifier will need to be a column in the query of your SQL trait. The column name in the query of the SQL trait should be the one that Identity Resolution uses to generate the identifier. 

### Are there any errors in the browser's Network or Console tab?

> When experiencing any issues saving the SQL Trait query or when Previewing the results of the SQL Trait query, open up the browser's Console and Network tabs to see if any errors occur upon clicking the Save/Preview buttons. If you find any errors, please copy the errors by expanding the error, take screenshots of the error, and share these details when creating a support ticket.

Reviewer: ```suggestion
If you experience issues saving the SQL Trait query or previewing the results of the SQL Trait query, open the browser's Console and Network tabs to see if any errors occurred upon clicking the Save/Preview buttons. If you find any errors, please expand the error and take a screenshot of it. You can then share these details when creating a support ticket.
```
